nemo/collections/asr/models/rnnt_models.py

```diff
@@ -204,13 +216,25 @@ def transcribe(
 
                 temporary_datalayer = self._setup_transcribe_dataloader(config)
                 for test_batch in tqdm(temporary_datalayer, desc="Transcribing"):
-                    encoded, encoded_len = self.forward(
+                    logits, logits_len = self.forward(
                         input_signal=test_batch[0].to(device), input_signal_length=test_batch[1].to(device)
                     )
-                    hypotheses += self.decoding.rnnt_decoder_predictions_tensor(
-                        encoded, encoded_len, return_hypotheses=return_hypotheses
-                    )
-                    del encoded
+                    if logprobs:
```

Comment:
This is incorrect - rnnt forward just does forward of encoder and nothing else. The logprobs of RNNT would be the joint matrix during inference - but we dont compute that. Revert this portion, its my mistake thinking RNNT logprobs could be calculated easily during inference.

Reply:
done.

Reply:
dropped.